tests/testthat/test-verb-count.R

```diff
@@ -4,13 +4,17 @@ test_that("generates expected SQL for common situations", {
   expect_snapshot(db %>% count(g))
   expect_snapshot(db %>% count(g, wt = x))
   expect_snapshot(db %>% count(g, sort = TRUE))
+
+  expect_snapshot(db %>% add_count(g, sort = TRUE))
```

Comment:
Out of scope here, just wondering:

```suggestion
  expect_snapshot({
    db %>% count(g)
    db %>% count(g, wt = x)
    db %>% count(g, sort = TRUE)
    db %>% add_count(g, sort = TRUE)
  })
```

?

Reply:
Neat, I didn't know this works resp. just didn't think about this pattern!

Reply:
It's not necessarily better: the existing code allows adding new snippets without having to accept the snapshot manually.